vunit/project.py

```diff
@@ -448,6 +438,17 @@ def _get_affected_files(self, target_files, get_depend_func):
             self._handle_circular_dependency(exc)
             raise CompileError
 
+    def _get_compile_order(self, dependency_graph):
```

Comment:
See [previous comment](https://github.com/VUnit/vunit/pull/551#discussion_r327845879) about merging `_get_compile_order` and `_get_affected_files`.